adapter/src/test/java/org/jvalue/ods/adapterservice/datasource/model/DatasourceConfig.json

```diff
@@ -21,5 +21,8 @@
     "license": "none",
     "displayName": "TestName",
     "description": "Describing..."
+  },
+  "schema": {
+    "data": "{a:1}"
```

Comment:
Can we do something like the following?
```json
"schema": {
  "a": 1
}
```

So there are real objects and not just a string directly under schema?